README.md

```diff
@@ -113,11 +113,76 @@ result = query("Write a report about xxx.") # Your question here
     <p> More details about VoyageAI: https://docs.voyageai.com/embeddings/ </p>
 </details>
 
+#### Vector Database Configuration
+<pre><code>config.set_embedding_config("vector_db", "(VectorDBName)", "(Arguments dict)")</code></pre>
+<p>The "VectorDBName" can be one of the following: ["Milvus"] (Under development)</p>
+<p> The "Arguments dict" is a dictionary that contains the necessary arguments for the Vector Database class.</p>
 
+<details>
+  <summary>Example (Milvus)</summary>
+    <pre><code>config.set_embedding_config("vector_db", "Milvus", {"uri": "./milvus.db", "token": ""})</code></pre>
```

Comment:
it should be `config.set_provider_config("vector_db", "Milvus", "(Arguments dict)")`